.travis.yml

```diff
@@ -18,9 +18,20 @@ before_install:
             sudo docker pull welder/web:latest
             sudo docker pull welder/web-with-coverage:latest
             sudo docker pull welder/web-e2e-tests:latest
-        else
+        fi
+        if [ "$COMMAND" == "cockpit-test" ]; then
+            sudo docker pull welder/web-e2e-tests:latest
+            sudo docker pull welder/web-cockpit:latest
+        fi
+        if [ "$COMMAND" != "end-to-end-test" ]; then
```

Comment:
`npm install` inside the (deleted) else block above, we don't need a second if statement with a negative condition to do that.

Reply:
updated.